ScanUnsupportedChars/ScanUnsupportedChars.ps1

```diff
@@ -726,6 +801,56 @@ if ($FilesWithInvalidCharsFixedName.Count -gt 0)
 
         Write-Host "***************************Items failed to rename table end*****************************" -ForegroundColor Yellow
     }
+    elseif($PSBoundParameters.ContainsKey('DestinationPath') -and !($PSBoundParameters.ContainsKey('RenameItems')))
+    {
+        Write-Host "========================== File COPY/MOVE start ==========================================" -ForegroundColor Yellow
+        Write-Host "Ensure UTF-8 Codepage and Encoding"
+        $PSDefaultParameterValues['*:Encoding'] = 'utf8'
+        CHCP 65001
+
+        # To prevent previous code from breaking, revert for copy to previous sharepath
+        $cleanSharePath = $SharePath
+        if($cleanSharePath.StartsWith("\\?\"))
+        {
+            $cleanSharePath = $cleanSharePath.Remove(0, 4)
+        }
+        if($cleanSharePath.StartsWith("\\?\unc\")){
+            $cleanSharePath = $cleanSharePath.Remove(0, 8)
+        }
+
+        foreach ($file in $FilesWithInvalidCharsFixedName) {
+
+            # is File or Folder already part of a synced directory, file can be skipped
+            $isHandled = $listFile.IsHandledThroughFolder($file.OriginalFilePath)
+            if(!$isHandled){
+                $root = [System.IO.Path]::GetPathRoot($file.OriginalFilePath);
+                $relative_path = $listFile.GetRelativePath($root, $file.OriginalFilePath); #[System.IO.Path]::GetRelativePath($root, $file.OriginalFilePath);
```

Comment:
why [System.IO.Path]::GetRelativePath() is not working ?

Reply:
This is not supported by .Net 4.7 **https://docs.microsoft.com/de-de/dotnet/api/system.io.path.getrelativepath?view=net-5.0&viewFallbackFrom=netframework-4.7** which is intended to be used - is there any .net framework version ensured when running azure file sync? Then I could use this as fallback.

Anyway the win32 API for GetRelativePath introduced again loss of utf-8 characters which is why I needed to reverse this implementation, by a new update and made a few tweeks to it. Tomorrow I'll have a final review to verify in larger context script is working as expected. If this is the case I'd merge the pull request.